### DISCUSSION TITLE
Kind of "Symmetric" patch sampling

### DISCUSSION THREAD

**Discussion by avKostanov** [TorchIO-project/torchio#700] [Q&A]
Hello, everyone. Thank you for a nice library. I have a question:

My research based on localization of ishemic stroke on very early stages. Currently, I'm working on an idea about compairing two brain hemishperes to distinguish stroke regions from healthy ones. So my question: is there any way to sample equal patches from both hemispheres? For ex: I have some images with segmented ASPECTS regions, and I would like compare symmetric pairs.

Thank you in advance!

**Comment by fepegar** [comment]
Hi, @avKostanov. Are your images symmetrical around the interhemispheric plane? You could maybe create a transform that samples some patches ([as demonstrated here](https://github.com/fepegar/torchio/discussions/692#discussioncomment-1528047)), and sample new patches using `Crop`. The patches locations are stored in the generated instance of `Subject`. Does that make sense?

**Reply by avKostanov** [reply]
Thank you very much for reply, @fepegar. Yes, I preprocessed data to symmetrical view, so hemispheres look similar. Previously, I've got an idea to figure this out by just loading similar image, flipped vertically. So patches with identical location from both images would give me symmetry, but CPU won't like it :D.  But, with your tips, now it makes more sense.
